### DISCUSSION TITLE
Some options are missing in Zorin OS 15.3

### DISCUSSION THREAD

**Discussion by Marcosffn** [xournalpp/xournalpp#3212] [Q&A]
Recently I moved from Ubuntu 20.04 to Zorin OS 15.3, and I lost some options that I had in Xournal++ in Ubuntu. In both situations, I have installed Xournal++ 1.1.0+ Dev/Instable version. I can't list all the missing options but some I used most frequently that I don't have access anymore in Zorin OS are the stabilization methods, send to back/backward or front/forward and to change the language. Some plugins also disappeared.
At the beginning I thought it was because of the incomplete translation to Portuguese, but when I executed Xournal with the terminal using LANG=en_US I still don't have access to these options.
Someone knows why this is happening and how to fix this problem?

**Comment by rolandlo** [comment]
Can you tell us, how you installed Xournal++ (.deb-installer, PPA, ... ) and can you post a screenshot of the version/git commit/compile date/gtk version table 
![grafik](https://user-images.githubusercontent.com/40485433/124821541-229cd700-df6f-11eb-85d3-71b32080556b.png)

from menu `Help > About`? It must be an older version that you have got on Zorin OS, I bet.

**Comment by Marcosffn** [comment]
![image](https://user-images.githubusercontent.com/67158009/124822768-b908d280-df46-11eb-8294-bd628ee46106.png)
Now I see that I installed an old version. I followed the tutorial on https://xournalpp.github.io/installation/linux/:
sudo add-apt-repository ppa:andreasbutti/xournalpp-master
sudo apt update
sudo apt install xournalpp
I think I should remove this version and install from github release page then.

**Comment by rolandlo** [comment]
Ok, the problem is that the Ubuntu 18.04 version doesn't compile on Launchpad (which provides the PPA packages) since August 2020 (due to some newer libraries we use).  Zorin 15.3 is based on Ubuntu 18.04, that's why you got that old version. 
You are right about the solution: remove that version and install from the github release page, either the [Bionic .deb binary](https://github.com/xournalpp/xournalpp/releases/download/nightly/xournalpp-1.1.0.dev-nightly.20210706-Ubuntu-bionic-x86_64.deb) or the [AppImage](https://github.com/xournalpp/xournalpp/releases/download/nightly/xournalpp-1.1.0.dev-nightly.20210706-x86_64.AppImage)

**Reply by Marcosffn** [reply]
Thank you for the support, I didn't know Zorin was based on Ubuntu 18.04.
